go/exocom-mock/exocom-mock.go

```diff
@@ -16,6 +15,7 @@ import (
 type ExoComMock struct {
 	ReceivedMessages []structs.Message
 	server           http.Server
+	serviceSockets   map[string]*websocket.Conn
```

Comment:
Do we really need infrastructure to store several websockets? Since this is only a mock Exocom for testing one particular ExoRelay instance, I only ever expect one websocket to connect to it.

So, can this just be:

```go
socket *websocket.Conn
```